server/config/config.go

```diff
@@ -768,7 +768,9 @@ func (c *ScheduleConfig) adjust(meta *configMetaData) error {
 	}
 	adjustFloat64(&c.LowSpaceRatio, defaultLowSpaceRatio)
 	adjustFloat64(&c.HighSpaceRatio, defaultHighSpaceRatio)
-	adjustSchedulers(&c.Schedulers, DefaultSchedulers)
+
+	// make a copy to avoid changing DefaultSchedules unexpectly
+	adjustSchedulers(&c.Schedulers, append(DefaultSchedulers[:0:0], DefaultSchedulers...))
```

Comment:
Would you like to add a test for this case or explain it more?

Reply:
Good idea. I'll add more description.

Reply:
I think clone and related comments should be inside `adjustSchedulers`.